doc/stdlib/base.rst

```diff
@@ -700,6 +700,22 @@ Types
        julia> f(apple)
        "I'm a FRUIT with value: 1"
 
+.. function:: @flagenum FlagEnum[::U] EnumValue1[=x] EnumValue2[=y]
```

Comment:
Should probably not use camel case for instances. Use lower case or upper case? Cf. https://github.com/JuliaLang/julia/pull/19506